charts/airbyte-server/templates/deployment.yaml

```diff
@@ -256,6 +258,10 @@ spec:
       {{ toYaml .Values.global.extraContainers | indent 8 }}
       {{- end }}
 
+<<<<<<< HEAD
```

Comment:
nit: probs not intended

Reply:
Strange, I did searched for it...
Thanks for noticing!